src/qt/forms/researcherwizardinvestorpage.ui

```diff
@@ -104,7 +104,7 @@
    <item>
     <widget class="QLabel" name="startOverLabel">
      <property name="text">
-      <string>Press &quot;start over&quot; to switch modes.</string>
```

Comment:
Can we change this to `To switch modes press "Start Over"` or `If you want to switch modes, press "Start Over"` or something similar? I almost read this as if this was a required step instead of an optional step. I imagine others might read it that way too

Reply:
I get your point, but perhaps we can change `Start Over` to something like `Restart` and just remove the label?

Reply:
I agree&mdash;it sounds like a call-to-action for a required step rather than a hint. 

@div72 You mean change the label to a button?

Reply:
I mean even just the positioning of the button conveys the idea that it'll take me back, does it need a hint?

Reply:
No, the hint is superfluous. Since that page has so much free space, though, it doesn't hurt anticipate that a user may wish to switch modes by providing a friendly suggestion.